plugins/webp-uploads/settings.php

```diff
@@ -86,7 +139,6 @@ function webp_uploads_media_setting_style(): void {
 	</style>
 	<?php
 }
-add_action( 'admin_head-options-media.php', 'webp_uploads_media_setting_style' );
```

Comment:
The `webp_uploads_media_setting_style` function is now no longer used. Should it be moved to `deprecated.php` with a `_deprecated_function()` call?

Reply:
ah, right. moved in f02c2d29ba33e019e847cd4bbe42c9a90b723705